### PR TITLE
Changes to move source types to output modules #2785

### PR DESCRIPTION
**Pull request by joachimmetz**
Changes to move source types to output modules #2785

### CODE REVIEW

File: data/sources.csv [hunk 1]
```diff
@@ -0,0 +1,192 @@
+data_type	short_source	source
```
Comment (rgayon) [inline data/sources.csv:1]: Is this a "CSV" when it's tab separated? :)

**Review by rgayon — APPROVED** [review]
Nit

File: data/sources.csv (hunk 1 above)
Comment (joachimmetz) [reply data/sources.csv:1]: what do you prefer? `sources.conf` instead?

Comment (joachimmetz) [reply data/sources.csv]: changed to `sources.config`

**Review by rgayon — APPROVED** [review]

**Pull request merged**